internal/examples/go.mod

```diff
@@ -8,10 +8,10 @@ require (
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/fx v1.10.0
 	go.uber.org/multierr v1.4.0
-	go.uber.org/thriftrw v1.22.0
+	go.uber.org/thriftrw v1.27.1-0.20210722163750-b1733d58df3c
 	go.uber.org/yarpc v1.42.1
 	go.uber.org/zap v1.13.0
-	google.golang.org/grpc v1.27.1
+	google.golang.org/grpc v1.28.0
```

Comment:
Why do we update `google.golang.org/grpc` here?

Reply:
`go get` did this - it looks like it's supposed to be 1.28.0 anyways?  This matches with what's at the root `go.mod`.

Reply:
Oh right this is in the internal/examples/go.mod. I thought this was the root one